src/component/FolderInfo.js

```diff
@@ -30,6 +30,13 @@ function FolderInfo() {
     getFolderInfo();
   }, []);
 
+  if (
+    !folderInfo.profileImage ||
+    !folderInfo.ownerName ||
+    !folderInfo.folderName
+  ) {
```

Comment:
이렇게 조건문이 길어질 경우 조건문을 해석해야하지만  아래처럼 임의의 변수에 할당하고 적절한 네이밍만 주어도 해석에 소모되는 리소스를 줄일 수 있습니다.

```suggestion
  if (isEmpty) {
```